src/lib/section.svelte

```diff
@@ -0,0 +1,34 @@
+<script>
+  export let isVisible = false;
+  export let title;
+</script>
+
+<div>
+  <h6 class="subtitle is-6 has-background-link-light is-unselectable pl-1">
+    <button
```

Comment:
I turned this into a button to make things slightly more accessible. This fixes a bunch of a11y-click-events-have-key-events errors.